src/components/tables/cells/fieldSelection/FieldActionButton.tsx

```diff
@@ -0,0 +1,63 @@
+import { Tooltip } from '@mui/material';
+import OutlinedToggleButton from 'components/shared/buttons/OutlinedToggleButton';
+import useOnFieldActionClick from 'hooks/fieldSelection/useOnFieldActionClick';
+import { useIntl } from 'react-intl';
+import { useFormStateStore_isIdle } from 'stores/FormState/hooks';
+import { getConstraintMessageId } from './shared';
+import { FieldActionButtonProps } from './types';
+
+export default function FieldActionButton({
+    bindingUUID,
+    constraint,
+    disabled,
+    field,
+    labelId,
+    selection,
+    tooltipPlacement,
```

Comment:
This is fine for now - but I think if we allow one prop we should probably just go ahead and allow all props to be passed in.

We'd still _probably_ spread it into the component so `title` could not be overwritten but that way we never have to update the component to expand capability.